_search-plugins/knn/approximate-knn.md

```diff
@@ -9,23 +9,26 @@ has_math: true
 
 # Approximate k-NN search
 
-The approximate k-NN search method uses nearest neighbor algorithms from *nmslib* and *faiss* to power
-k-NN search. To see the algorithms that the plugin currently supports, check out the [k-NN Index documentation]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index#method-definitions).
-In this case, approximate means that for a given search, the neighbors returned are an estimate of the true k-nearest neighbors. Of the three search methods the plugin provides, this method offers the best search scalability for large data sets. Generally speaking, once the data set gets into the hundreds of thousands of vectors, this approach is preferred.
+Standard k-NN search methods compute similarity using a “brute-force” approach that measures nearest distance between a query and a number of points, which produces exact results. This works well in many applications. However, in the case of extremely large data sets with high dimensionality, this creates a scaling problem that reduces the efficiency of the search. Approximate k-NN search methods can overcome this by employing tools that restructure indexes more efficiently and reduce the dimensionality of searchable vectors. Using this approach requires a sacrifice in accuracy but increases search processing speeds appreciably.
 
-The k-NN plugin builds a native library index of the vectors for each "knn-vector field"/ "Lucene segment" pair during indexing that can be used to efficiently find the k-nearest neighbors to a query vector during search. To learn more about Lucene segments, see the [Apache Lucene documentation](https://lucene.apache.org/core/8_9_0/core/org/apache/lucene/codecs/lucene87/package-summary.html#package.description).
-These native library indices are loaded into native memory during search and managed by a cache. To learn more about
-pre-loading native library indices into memory, refer to the [warmup API]({{site.url}}{{site.baseurl}}/search-plugins/knn/api#warmup-operation). Additionally, you can see what native library indices are already loaded in memory, which you can learn more about in the [stats API section]({{site.url}}{{site.baseurl}}/search-plugins/knn/api#stats).
+The approximate k-NN search methods leveraged by OpenSearch use approximate nearest neighbor (ANN) algorithms from nmslib, faiss, and Lucene libraries to power k-NN search. These search methods employ ANN to improve search latency over large data sets. Of the three search methods the k-NN plugin provides, this method offers the best search scalability for large data sets. This approach is the preferred method when a data set reaches hundreds of thousands of vectors.
```

Comment:
Could we add links to [nmslib](https://github.com/nmslib/nmslib), [faiss](https://github.com/facebookresearch/faiss), and [Lucene](https://lucene.apache.org/)?